cmd/monoctl/update/cluster.go

```diff
@@ -65,16 +67,28 @@ func NewUpdateClusterCmd() *cobra.Command {
 				}
 			}
 
+			if newName != "" && newName != name {
+				prompt := promptui.Prompt{
```

Comment:
can you please wrap it like [for example](https://github.com/finleap-connect/monoctl/blob/ff804fdf96fafd715830c6408a5ca7f48c899e72/internal/prompt/prompt.go#L15) or was it intentional to not do so?

Reply:
updated